circle.yml

```diff
@@ -1,13 +1,24 @@
 machine:
   xcode:
-    version: "6.3.1"
+    version: "8.1.0"
+  environment:
+    FASTLANE_SKIP_UPDATE_CHECK: true
+    BUNDLE_PATH: vendor/bundle
+    UPLOAD_IOS_SNAPSHOT_BUCKET_NAME: eidolon-ci
 dependencies:
+  cache_directories:
+    - vendor/bundle
+    - ~/.cocoapods/repos
   override:
```

Comment:
That’s def ace 👌 but I’m less worried about the time over just installation working at all.